### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD018.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#23] [General]
# https://blog.3place.cn/3place/fsd/FSD018.html

https://blog.3place.cn/3place/fsd/FSD018.html

**Comment by picklecai** [comment]
我失去安宁
内心沉闷
要找回安宁
永远不能

这是不可逆的过程。

**Comment by picklecai** [comment]
但愿让
紧紧抱住他
让我吻他
吻个酣畅
受到他亲吻
死也无妨！

朝闻道，夕死可矣。
